core/README.md

```diff
@@ -0,0 +1,31 @@
+# Chainlink Core
+
+This directory contains the majority of the code for the backend of Chainlink.
+
+Outside of the code contained in this repo, some static assets are pulled in using
+[packr](https://github.com/gobuffalo/packr), in the sibling directory `packr/`.
+For the backend, the solidity ABIs are used to generate types for interacting with
+Ethereum contracts. Additionally, the static assets generated by compiling the
+sibling directroy `operator-ui/` are built by packr and included in the final
+binary.
+
+See the [root README](https://github.com/smartcontractkit/chainlink/blob/master/README.md#install)
+for instructions on how to build the full Chainlink node.
+
+## Common Commands
+
+**Install:**
+
+By default `go install` will install this directory under the name `core`.
+You can instead, build it, and place it in your path as `chainlink`:
+```
+go build -o chainlink-core .
+cp chainlink-core $GOPATH/bin/chainlink
+rm ./chainlink-core
+```
+
+**Test:**
+
+The golang testsuite is almost entirely parallelizable, and so running the default
+`go test ./...` will commonly peg your processor. Limit parallelization with the
+`-p 2` or whatever best fits your computer: `go test -p 4 ./...`.
```

Comment:
Too much explanation here.  I'd suggest saying how to test, then rationalize the options after.  The people reading this are likely more interested in how to test than why its done that way.

Reply:
Maybe default to a stable option?  Like
```
# A higher parallel number can speed up tests at the expense of more RAM.
go test -parallel=1 ./...
```

We could remove our comment on the main README about golang testing there too to avoid duplication.

Reply:
Went with this example to lead the testing section with an explanation below it. I'm leaving the main README for now, because I think it's more discoverable, but open to changing it in the future.